requirements-dev.txt

```diff
@@ -4,9 +4,8 @@ ipdb==0.13.9
 pytest==7.0.1
 pytest-cov==3.0.0
 pytest-sugar==0.9.4
-PyMySQL>=1.0.0,<=1.0.2
-sphinx>=1.8.1, <4.4.1
+PyMySQL==1.0.2
+Sphinx>=4.0
 sphinxcontrib-asyncio==0.3.0
-sqlalchemy>1.2.12,<=1.3.24
+sqlalchemy==1.3.24
```

Comment:
Maybe move the unrelated mangling with pins to a separate PR too?

Reply:
removed, partially split to #740